HLTrigger/JetMET/plugins/HLTScoutingPFProducer.cc

```diff
@@ -0,0 +1,249 @@
+// -*- C++ -*-
+//
+// Package:    HLTrigger/JetMET
+// Class:      HLTScoutingPFProducer
+// 
+/**\class HLTScoutingPFProducer HLTScoutingPFProducer.cc HLTrigger/JetMET/plugins/HLTScoutingPFProducer.cc
+
+Description: Producer for ScoutingPFJets from reco::PFJet objects, ScoutingVertexs from reco::Vertexs and ScoutingParticles from reco::PFCandidates
+
+*/
+//
+// Original Author:  Dustin James Anderson
+//         Created:  Fri, 12 Jun 2015 15:49:20 GMT
+//
+//
+
+// system include files
+#include <memory>
+
+// user include files
+#include "FWCore/Framework/interface/Frameworkfwd.h"
+#include "FWCore/Framework/interface/stream/EDProducer.h"
+#include "FWCore/Framework/interface/Event.h"
+#include "FWCore/Framework/interface/MakerMacros.h"
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+
+#include "DataFormats/JetReco/interface/PFJet.h"
+#include "DataFormats/METReco/interface/MET.h"
+#include "DataFormats/METReco/interface/METCollection.h"
+#include "DataFormats/ParticleFlowCandidate/interface/PFCandidateFwd.h"
+#include "DataFormats/ParticleFlowCandidate/interface/PFCandidate.h"
+#include "DataFormats/VertexReco/interface/Vertex.h"
+#include "DataFormats/VertexReco/interface/VertexFwd.h"
+#include "DataFormats/BTauReco/interface/JetTag.h"
+
+#include "DataFormats/Scouting/interface/ScoutingPFJet.h"
+#include "DataFormats/Scouting/interface/ScoutingParticle.h"
+#include "DataFormats/Scouting/interface/ScoutingVertex.h"
+
+#include "DataFormats/Math/interface/deltaR.h"
+
+class HLTScoutingPFProducer : public edm::stream::EDProducer<> {
```

Comment:
I did not try to do it, but I'm guessing this could safely be a `::global` producer instead of a `::stream` producer ?